_snippets/integrations/creating-nodes/testing.md

```diff
@@ -18,8 +18,8 @@ You can test your node as you build it by running it in a local n8n instance.
   ```
 
     !!! note "Check your directory"
-        Make sure you run `npm link <node-name>` in the nodes directory within your n8n installation. This is probably something like `~/.n8n/nodes/`
-    
+        Make sure you run `npm link <node-name>` in the nodes directory within your n8n installation. This is probably something like `~/.n8n/custom/`
```

Comment:
```suggestion
        Make sure you run `npm link <node-name>` in the nodes directory within your n8n installation. This can be: 
        * `~/.n8n/custom/`
        * `~/.n8n/<your-custom-name>`: if your n8n installation set a different name using `N8N_CUSTOM_EXTENSIONS`.
```